docs/bis/ec/ec-format.md

```diff
@@ -173,3 +178,22 @@ This format does not specify any units so it may only be used in a [format overr
       ]
     }
 ```
+
+This format specifies units which have labels that are intentionally missing or set to an empty string.
+
+```json
+    "MyMetersFormat": {
+      "schemaItemType": "Format",
+      "label": "real",
+      "type": "Decimal",
+      "precision": 4,
+      "formatTraits": [
+        "KeepSingleZero",
+        "KeepDecimalPoint",
+        "ShowUnitLabel"
+      ],
+      "composite": {
+        "units": [{ "name": "Units.KM" }, { "name": "Units.M", "label": "" }, { "name": "Units.CM", "label": "centimeter" }, { "name": "Units.MM", "label": "millimeter" }]
```

Comment:
@ColinKerr : this has the label for CM as meter ... I know it doesn't really matter but it'd be better if it was correct

Done !